eng/.docsettings.yml

```diff
@@ -138,6 +138,7 @@ known_content_issues:
   - ['sdk/keyvault/README.md', '#3113']
   - ['sdk/keyvault/azure-security-keyvault-administration/README.md', '#3113']
   - ['sdk/loganalytics/microsoft-azure-loganalytics/README.md', '#3113']
+  - ['sdk/monitorexporter/azure-analytics-monitorexporter/swagger/README.md', '#3113' ]
```

Comment:
Need to add a suppression for the Swagger file as well.

Reply:
This is the suppression for the swagger file. @alzimmermsft what other swagger file are you referring to?

Reply:
Misread this line, thought this was for the README itself.

Reply:
for beta.1 we landed on opentelemetry-exporter-azuremonitor to follow the convention used by all the OT exporters, since this isn't a traditional client SDK.